data-prepper-logstash-configuration/src/test/resources/org/opensearch/dataprepper/logstash/mapping/amazon_es.mapping.yaml

```diff
@@ -0,0 +1,8 @@
+pluginName: opensearch
```

Comment:
Minor: Would it be clearer if the file name is something like:  logstash_to_dataprepper_config.mapping.yaml
Minor: it would be nicer to cover more valid configuration mappings in the unit tests, such as username, password, and many more. 

It's unit test. I am OK with you adding more tests later. I will approve the pull request now.

Reply:
Thanks, I'll will add more tests later